src/components/InputControl/InputControl.style.js

```diff
@@ -0,0 +1,24 @@
+import styled from 'styled-components'
+
+export const container = styled.div`
```

Comment:
Can you please name styled components starting with a capital letter like `Container`